modules/ROOT/pages/acm-lightning-components.adoc

```diff
@@ -0,0 +1,240 @@
+
+Members can request access for an existing client application and API instance, or request to create a new client application by specifying the application name and description and choosing an API instance.
+
+* *API*: Choose the API.
+* *Button Border Radius*: Set the radius of the button border. A larger radius makes the button look more rounded.
+* *Button Align*: Set button alignment to left, right, or center.
+* *Show Modal Title Line*: Show the title line in the access request modal.
+// check description of Show Modal Title Line
+
+== API Card
+
+The API card component shows information about an API, including icon, name, and description.
+
+* *API*: Choose the API to show.
+* *Override URL*: When the user clicks the *API Details* button on the card, a URL opens. If this optional field is set, the URL in the field overrides the default URL. This could be used to direct users to a documentation or marketing page about the API, which would also have a link to the API's main page.
+* *Show Buttons*: Show the *API Details* button. If the button is not shown, users can click anywhere on the card to show the API details.
+* *Card Height*: Set the card height.
+* *Card Margin*: Set the amount of margin space around the card.
+* *Card Border*: Set the thickness of the card borders.
+* *Card Border Radius*: Set the radius of the card borders. A larger radius makes the cards look more rounded.
+* *Card Shadow*: Set the shadow beneath the card.
+
+== API Carousel
+
+The API carousel component shows a horizontally scrollable set of API cards.
+
+* *Show Buttons*: Show the *API Details* button on each card. If the button is not shown, users can click anywhere on the card to show the API details.
+* *Max Cards Display*: Set the maximum number of cards to show.
+* *Rows*: Set the number of rows of cards in the carousel.
+* *Show Arrows*: Show arrows on the right and left sides of the carousel. Clicking the arrows scrolls through the cards.
+* *Arrow Size*: Set the size of the arrows on the right and left sides.
+* *Show Dots*: Show dots below the cards. Clicking the dots scrolls through the cards.
+* *Card Height*: Set the card height.
+* *Card Margin*: Set the amount of margin space around each card.
+* *Card Border*: Set the thickness of the card borders.
+* *Card Border Radius*: Set the radius of the card borders. A larger radius makes the cards look more rounded.
+* *Card Shadow*: Set the shadow beneath each card.
+* *Message If Empty*: Show this message if the carousel has no APIs. Default is "There are no APIs available right now."
+* *Filter Condition (Advanced)*: Filter APIs using package names, field names such as "API name" or "API description", and comparison of strings to those fields.
+// need better info on using Filter condition
+
+== API Console
+
+The interactive API console component shows information about your APIs and their methods, and allows users to test the methods by sending requests.
```

Comment:
allows > enables

Reply:
Good point! Changed this and the other instances of "allow" in the file to "enable."